sympy/functions/elementary/tests/test_piecewise.py

```diff
@@ -491,7 +501,36 @@ def test_as_expr_set_pairs():
         [((x - 2)**2, Interval(0, oo)), (0, Interval(-oo, 0, True, True))]
 
 
+@XFAIL
 def test_S_srepr_is_identity():
+    # this fails as follows:
```

Comment:
This is probably caused by `ExprCondPair` not being defined globally. Therefore `sympify` will interpret it as a symbol.
```
>>> from sympy.functions.elementary.piecewise import ExprCondPair
>>> d = {'ExprCondPair': ExprCondPair}
>>> s
"Piecewise(ExprCondPair(Integer(10), Equality(Symbol('x'), Integer(0))), ExprCondPair(Integer(12), S.true))"
>>> S(s, locals=d)
Piecewise((10, Eq(x, 0)), 12)
```